gpu_benchmark.cpp

```diff
@@ -43,6 +43,9 @@ cublasHandle_t get_blas_handle() {
 
     if(!is_initialized) {
         cublasCreate(&cublas_handle);
+        #ifdef USE_TF32_CUBLAS
```

Comment:
does `USE_TF32_CUBLAS` require that we are using `value_type=float`?

Reply:
Yes, otherwise tensor cores are not used it seems.